### PR TITLE
10.0 project timesheet currency

### PR DESCRIPTION
**Pull request by damdam-s**
superseed #272

### CODE REVIEW

**Review by gurneyalex — APPROVED** [review]

File: project_timesheet_currency/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# Copyright 2017 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Project Timesheet Currency',
+    'summary': """
+        Multi-currency analytic costs""",
+    'version': '10.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': 'Camptocamp SA,Odoo Community Association (OCA)',
```
Comment (guewen) [inline project_timesheet_currency/__manifest__.py]: s/Camptocamp SA/Camptocamp/

File: project_timesheet_currency/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    'website': 'http://www.camptocamp.com',
```
Comment (guewen) [inline project_timesheet_currency/__manifest__.py]: s/http/https/

File: project_timesheet_currency/models/analytic.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+        res = super(AccountAnalyticLine, self)._get_timesheet_cost(values)
+        project_id = values.get('project_id', False)
+        if project_id and res.get('amount', False):
+            project_model = self.env['project.project']
+            project = project_model.browse(project_id)
+            currency = project.currency_id
+            base_currency = self.env.user.company_id.currency_id
+            res['amount_currency'] = res.get('amount')
+            res['amount'] = currency.compute(
+                res.get('amount'), base_currency)
```
Comment (guewen) [inline project_timesheet_currency/models/analytic.py]: at this point you know you have an amount key so I would write `res['amount']`

File: project_timesheet_currency/models/analytic.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+from odoo import models
+
+
+class AccountAnalyticLine(models.Model):
+    _inherit = 'account.analytic.line'
+
+    def _get_timesheet_cost(self, values):
+        res = super(AccountAnalyticLine, self)._get_timesheet_cost(values)
+        project_id = values.get('project_id', False)
```
Comment (guewen) [inline project_timesheet_currency/models/analytic.py]: `, False` is useless

**Review by guewen — APPROVED** [review]
I changed the details of my review

**Pull request merged**